dokka-integration-tests/gradle.properties

```diff
@@ -10,3 +10,7 @@ version=2.0.20-SNAPSHOT
 org.jetbrains.dokka.javaToolchain.mainCompiler=8
 org.jetbrains.dokka.javaToolchain.testLauncher=8
 org.jetbrains.dokka.integration_test.parallelism=2
+
+# !!! ATTENTION: do not commit this !!!
+# Uncomment next line to skip running Dokka integration tests
+#org.jetbrains.dokka.integration_test.skip=true
```

Comment:
FYI: @atyrin / @vmishenev after merging this PR you can uncomment this line locally to not run integration tests when running `build`/`check` tasks